src/app/rpmostree-builtin-rollback.c

```diff
@@ -80,6 +80,7 @@ rpmostree_builtin_rollback (int             argc,
     rpmostree_get_options_variant (opt_reboot,
                                    FALSE,  /* allow-downgrade */
                                    FALSE,  /* cache-only */
```

Comment:
Maybe we should be setting `cache-only` here?

Reply:
Though it has no purpose at all in this case, right? In fact, we're really just abusing `rpmostree_get_options_variant` since none of these options (apart from `reboot`) are actually supported by the D-Bus method. I added a comment to clarify that!